src/components/Settings/Settings.js

```diff
@@ -26,22 +40,119 @@ export default class Settings extends React.Component {
             value={this.props.currentNetwork}
             onChange={this.handleChangeSelectedNetwork}
           >
-            <option value="MainNet">MainNet</option>
-            <option value="TestNet">TestNet</option>
-            <option value="CozNet">CozNet</option>
-            <option value="nOSLocal">nOS Local</option>
+            {map(settings.networks, this.renderNetworkOption)}
           </select>
         </label>
+        <div className={styles.networkDetails}>
+          Custom Neoscan URL: {this.getCurrentNetworkUrl()}
```

Comment:
I've added this back because now we are storing network name and the bottom part of the app is showing the name instead of url.